bindings/pyroot/pythonizations/src/CPPInstancePyz.cxx

```diff
@@ -95,10 +91,10 @@ PyObject *op_reduce(CPPInstance *self, PyObject * /*args*/)
       static TBufferFile s_buff(TBuffer::kWrite);
       s_buff.Reset();
       // to delete
-      if (s_buff.WriteObjectAny(self->GetObject(),
+      if (s_buff.WriteObjectAny(CPyCppyy::Instance_AsVoidPtr(self),
                                 TClass::GetClass(className.c_str())) != 1) {
          PyErr_Format(PyExc_IOError, "could not stream object of type %s",
-                      Cppyy::GetScopedFinalName(self->ObjectIsA()).c_str());
+                      GetScopedFinalNameFromPyObject(self).c_str());
```

Comment:
Just for my knowledge, this function in particular is defined at global scope, whereas others such as `Instance_AsVoidPtr` above are inside a namespace?

Reply:
Because this one is not part of CPyCppyy. It is part of our own [PyzCppHelpers.hxx](https://github.com/root-project/root/pull/14414/files#diff-4c25ee634d30884a5fa329b7a3b99de2299e1b0c095de94bb4a43bba036faa72) functions